modules/4_nodes/nodes.tf

```diff
@@ -66,7 +66,7 @@ resource "openstack_compute_instance_v2" "bootstrap" {
     name        = "${var.cluster_id}-bootstrap"
     flavor_id   = var.scg_id == "" ? data.openstack_compute_flavor_v2.bootstrap.id : openstack_compute_flavor_v2.bootstrap_scg[0].id
     image_id    = var.bootstrap["image_id"]
-    availability_zone   = var.openstack_availability_zone
+    availability_zone   = lookup(var.bootstrap, "availability_zone", "") == "" ? var.openstack_availability_zone : var.bootstrap["availability_zone"]
```

Comment:
These conditions becomes simpler when we look for `is defined`.
```
availability_zone   = lookup(var.bootstrap, "availability_zone", var.openstack_availability_zone)
```

Reply:
Done.